server/routes/auth.js

```diff
@@ -1,73 +1,13 @@
 const express = require('express');
-const { path, pathOr } = require('ramda');
 
-const createAuthRouter = ({
-  logger,
-  formParser,
-  authenticateUser,
-  createUserSession,
-}) => {
+const createAuthRouter = ({ signIn, signInCallback, signOut }) => {
```

Comment:
Only thought is logging since the logger is passed in on creation. But we can do that in a later PR

Reply:
We have a ticket on the sprint backlog for that IIRC